roaring/_util.pony

```diff
@@ -0,0 +1,10 @@
+primitive _Bits 
+  """
+  Primitive for having a single source of truth for when we un-assume endianness
+  """
+
+  fun mostsig(value: U32): U16 => (value >> (value.bitwidth() / 2)).u16()
```

Comment:
it is pony style guide recommendation to name variables in snake case, so this should be `most_sig`. See: https://github.com/ponylang/ponyc/blob/main/STYLE_GUIDE.md#naming